### PR TITLE
Fixes #6 implements images and their pagination, also added susper icon

### LINKED ISSUE #6: implement image search in susper
present susper susper-backbone has a choice to search images, this could be implemented here.

### PR DESCRIPTION
**Pull request by Marauderer97**
@nikhilrayaprolu Please review. Added image functionality.

### CODE REVIEW

**Comment by Marauderer97** [comment]
Screenshot: image

File: src/app/results/results.component.html [hunk 1]
```diff
@@ -1,39 +1,53 @@
...
-          <a href='{{item.link}}'>{{item.link}}</a>
-        </div>
-        <div>
-          {{item.pubDate|date:'fullDate'}}
-        </div>
+<div class="row">
+  <div class="col-md-1">
+    <p>SIDEBAR</p>
+  </div>
+  <div class="col-md-11">
```
Comment (nikhilrayaprolu) [inline src/app/results/results.component.html]: please indent these lines.

File: src/app/results/results.component.html [hunk 2]
```diff
@@ -1,39 +1,53 @@
...
+    <nav aria-label="Page navigation">
+      <ul class="pagination">
+        <li class="page-item"><span class="page-link" href="#" (click)="decPresentPage()">Previous</span></li>
+        <li class="page-item" *ngFor="let num of getNumber(maxPage)"><span class="page-link" *ngIf="presentPage>=4 && num<=noOfPages" (click)="getPresentPage(presentPage-3+num)" href="#">{{presentPage-3+num}}</span>
+          <span class="page-link" *ngIf="presentPage<4 && num<=noOfPages" (click)="getPresentPage(num+1)" href="#">{{num+1}}</span></li>
+        <li class="page-item"><span class="page-link" (click)="incPresentPage()">Next</span></li>
+      </ul>
+    </nav>
   </div>
 </div>
```
Comment (nikhilrayaprolu) [inline src/app/results/results.component.html]: good work @Marauderer97 but there are some minor indentation issues, please resolve. these.

File: src/app/results/results.component.html [hunk 3]
```diff
@@ -1,39 +1,53 @@
...
-        <div>
-          {{item.pubDate|date:'fullDate'}}
-        </div>
+<div class="row">
+  <div class="col-md-1">
+    <p>SIDEBAR</p>
+  </div>
+  <div class="col-md-11">
+    <div class="btn-group">
+    <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown">
```
Comment (nikhilrayaprolu) [inline src/app/results/results.component.html]: please indent these lines

File: src/app/results/results.component.html [hunk 4]
```diff
@@ -1,39 +1,53 @@
...
+      <ul class="pagination">
+        <li class="page-item"><span class="page-link" href="#" (click)="decPresentPage()">Previous</span></li>
+        <li class="page-item" *ngFor="let num of getNumber(maxPage)"><span class="page-link" *ngIf="presentPage>=4 && num<=noOfPages" (click)="getPresentPage(presentPage-3+num)" href="#">{{presentPage-3+num}}</span>
+          <span class="page-link" *ngIf="presentPage<4 && num<=noOfPages" (click)="getPresentPage(num+1)" href="#">{{num+1}}</span></li>
+        <li class="page-item"><span class="page-link" (click)="incPresentPage()">Next</span></li>
+      </ul>
+    </nav>
   </div>
 </div>
+</div>
```
Comment (nikhilrayaprolu) [inline src/app/results/results.component.html]: good work @Marauderer97 but there are some minor indentation issues, please resolve. these.

**Comment by Marauderer97** [comment]
@nikhilrayaprolu done! :+1:

**Pull request closed without merging**